doc/recognition.xml

```diff
@@ -195,19 +195,19 @@ A recognition info record is a &GAP; component object. It is a member
 of the family
 
 <ManSection>
-<Fam Name="RecognitionInfoFamily"/>
+<Fam Name="RecogNodeFamily"/>
 </ManSection>
 
 and is in the category
 
 <ManSection>
-    <Filt Name="IsRecognitionInfo" Type="Category"/>
+    <Filt Name="IsRecogNode" Type="Category"/>
 </ManSection>
 
 and is <Ref Filt="IsAttributeStoringRep" BookName="Ref"/>, such that we can
 define attributes for it, the values of which are stored once they are
 known. A recognition info record always represents a whole binary tree of such
-records, see the attributes <Ref Attr="RIFac"/> and <Ref Attr="RIKer"/>
+records, see the attributes <Ref Attr="ImageNode"/> and <Ref Attr="KernelNode"/>
```

Comment:
I don't like that that these names do not reference the fact that this is for recognition. If I here "image node" then I don't know what this is about at all.

Elsewhere in this PR you introduce  `IsRecogNode`. So how about calling these `ImageRecogNode` or `RecogNodeForImage` or so (and similar for the kernel)?

Reply:
I like `ImageRecogNode`. 👍